app/assets/stylesheets/wcag.scss

```diff
@@ -195,10 +195,6 @@ html {
     color: var(--primary-medium);
   }
 
-  .badge-notification.clicks {
-    color: var(--primary-high);
```

Comment:
Can we keep this color override for the pseudo-element? It's likely that the default styling doesn't have enough contrast.